monitors/tests/test_new_end_of_run_monitor.py

```diff
@@ -0,0 +1,215 @@
+"""
+Unit tests for the end of run monitor
+"""
+import unittest
+import os
+import json
+import csv
+from filelock import FileLock
+from mock import (Mock, patch, call)
+
+from utils.clients.queue_client import QueueClient
+from monitors.settings import (CYCLE_FOLDER, LAST_RUNS_CSV)
+from monitors.new_end_of_run_monitor import (InstrumentMonitor,
+                                             get_prefix_zeros,
+                                             FileNotFoundError,
+                                             InstrumentMonitorError,
+                                             update_last_runs,
+                                             main)
+
+# Test data
+SUMMARY_FILE = ("WIS44731Hayden,Waite,"
```

Comment:
Can you change the names on this test data - Assuming this is actually a real person right now?